experimenter/experimenter/experiments/constants.py

```diff
@@ -54,9 +59,17 @@ class ApplicationConfig:
     slug: str
     app_name: str
     channel_app_id: Dict[str, str]
-    kinto_collection: str
+    default_kinto_collection: str
     randomization_unit: str
     is_web: bool
+    additional_kinto_collections: list[str] = field(default_factory=list)
```

Comment:
Won't we want this to be a dict of feature_config slugs to collection ids?

Reply:
Also if we're gonna introduce this field here in this pr, do we wanna also setup the logic that populates and consumes it?